terraform/gitops/k8s-cluster-config/app-deploy.tf

```diff
@@ -200,6 +200,11 @@ locals {
   pm4ml_keycloak_realm_env_secret_map = { for key, pm4ml in local.pm4ml_var_map :
     "${var.pm4ml_oidc_client_secret_secret}-${key}" => var.pm4ml_oidc_client_secret_secret_key
   }
+
+  pm4ml_provider_secret_map = { for key, pm4ml in local.pm4ml_var_map :
```

Comment:
any reason why this is not just added to the same map as pm4ml_keycloak_realm_env_secret_map?

Reply:
The only reason is that I do not know the syntax to do this.

Reply:
ok, let's leave it as is and then clean up if necessary, especially if we end up removing the other client.